src/Controller/LabsController.php

```diff
@@ -99,6 +100,10 @@ public function postAction(Request $request, string $id) : Response
         $arguments['blocks'] = $arguments['post']
             ->then($this->willConvertContent());
 
+        if ($this->isGranted('FEATURE_CAN_USE_HYPOTHESIS')) {
+            $arguments['hypothesisOpener'] = new HypothesisOpener();
+        }
```

Comment:
this is duplicated between controllers, but there's no clear way yet of removing it, 3 lines are not a problem

Reply:
and they model two necessary decisions: how to call the ViewModel class and which feature flag to check, so they don't contain boilerplate